src/transport/SessionManager.h

```diff
@@ -126,6 +126,8 @@ class DLL_EXPORT SessionManager : public TransportMgrDelegate, public FabricTabl
     SessionManager();
     ~SessionManager() override;
 
+    FabricTable * GetFabricTable() const { return mFabricTable; }
```

Comment:
This is a really weird place to expose the fabric table.

Reply:
Didn't see the mFabricTable in the InteractionModelEngine. Will use that.